scrappers/spiders/ELDEMOCRATA_spider.py

```diff
@@ -0,0 +1,135 @@
+import logging
+
+import scrapy
+from scrapy.http.response.html import HtmlResponse
+
+logging.basicConfig(filename="run.log", encoding="utf-8", level=logging.DEBUG)
+
+
+def get_title(response: HtmlResponse) -> str:
+    return response.xpath('//*[contains(@class, "entry-title")]//text()').get()
+
+
+def get_epigraph(response: HtmlResponse) -> str:
+    return response.xpath(
+        '//*[contains(@class, "newsever-categories category-color-1")]//text()'
+    ).get()
+
+
+def get_date(response: HtmlResponse) -> str:
+    try:
+        date = response.xpath(
+            '//div[contains(@class, "post-meta-detail")]'
+            '/span[contains(@class, "item-metadata posts-date")]//text()'
+        ).getall()[-1]
+        return date
+    except Exception as e:
+        logging.error(f"It was not possible to obtain the date. Error: {e}")
+        return None
+
+
+def get_authors(response: HtmlResponse) -> list[str]:
+    return response.xpath(
+        '//div[contains(@class, "post-meta-detail")]'
+        '/span[contains(@class, "entry-meta")]/span/a/text()'
+    ).getall()
+
+
+def get_authors_links(response: HtmlResponse) -> list[str]:
+    return response.xpath(
+        '//div[contains(@class, "post-meta-detail")]'
+        '/span[contains(@class, "entry-meta")]/span/a/@href'
+    ).getall()
+
+
+def get_comments(response: HtmlResponse) -> list[dict[str, str]]:
+    return [None]
+
+
+def get_excerpt(response: HtmlResponse) -> str:
+    return None
+
+
+def get_content(response: HtmlResponse) -> str:
+
+    content = " ".join(
+        response.xpath(
+            '//div[contains(@class, "entry-content read-details")]'
+            "/div[2]"
+            "/*[self::p or self::h2 or self::h3 or self::h4 or self::h5 or self::h6]"
+            "//text()"
+        ).getall()
+    )
+
+    titles = response.xpath(
+        '//*[contains(@class, "entry-content read-details")]'
+        "//*[self::h3 or self::h4 or self::h5 or self::h6]//text()"
+    ).getall()
+
+    for title in titles:
+        content = content.replace(title, title + ".")
+
+    return content
```

Comment:
Limpieza propuesta más arriba: 

```suggestion
    content = remove_social_networks(response, content)
    cleaned_content = clean_string(content, ["string x reemplazar 1", "string x reemplazar 2", ...])
    return cleaned_content
```